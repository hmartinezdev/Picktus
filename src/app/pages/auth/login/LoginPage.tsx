import Login from '@containers/Login';
import React from 'react';
import { IPageProps } from '../../page.type';

const LoginPage = (props: IPageProps): React.ReactElement<Login> => <Login {...props} />;

LoginPage.displayName = 'LoginPage';

export default LoginPage;
