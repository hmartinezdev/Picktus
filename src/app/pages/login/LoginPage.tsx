import Login from '@containers/Login';
import React from 'react';
import { IPageProps } from '../page.type';

export default (props: IPageProps): React.ReactElement<Login> => <Login {...props} />;
