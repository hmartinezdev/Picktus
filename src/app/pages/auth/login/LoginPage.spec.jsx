import { shallow } from 'enzyme';
import React from 'react';
import LoginPage from './LoginPage';

let wrapper;

const setup = () => shallow(<LoginPage />);

describe('<LoginPage />', () => {
  test('should render properly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
