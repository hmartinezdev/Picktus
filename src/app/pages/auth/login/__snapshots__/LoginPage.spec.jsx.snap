// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginPage /> should render properly 1`] = `
<Authentication>
  <ConnectFunction
    key="LoginHandler"
  />
</Authentication>
`;
